addons/beehave/nodes/beehave_tree.gd

```diff
@@ -291,3 +291,15 @@ func _get_debugger_data(node: Node) -> Dictionary:
 
 func get_class_name() -> Array[StringName]:
 	return [&"BeehaveTree"]
+	
+
+# required to avoid lifecycle issues on initial load
+# due to loading order problems with autoloads
+func _get_global_metrics() -> Node:
```

Comment:
Autoloads will be added to the scene tree, so we can look them up dynamically. Unfortunately, referencing them directly may not work, as the global class cache may not have loaded them at the time this class here is parsed.